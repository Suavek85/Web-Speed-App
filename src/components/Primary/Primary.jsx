import React from 'react'
import classNames from 'classnames/bind'
import { useMediaQuery } from 'react-responsive'

import InputWrapper from './Input/InputWrapper'
import LighthouseWrapper from './Lighthouse/LighthouseWrapper'
import { tabletWidth } from'../../scss/mediaqueries'
import styles from './Primary.scss'

export default function Primary() {

  //STYLES
  const cx = classNames.bind(styles)
  const getClass = cx({
    block: true,
    blockTablet: useMediaQuery(tabletWidth),
  })

  return (
    <div className={ getClass } >
      <div className={ styles.blockInput } >
        <InputWrapper />
      </div>     
      <LighthouseWrapper />
    </div>
    
  )
}
