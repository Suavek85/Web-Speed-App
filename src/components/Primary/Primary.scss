@import '../../scss/styles';

$block: primary;
$blockTablet: modifier($block, tablet);
$blockInput: elem($block, input);

.primary {
  display: flex;
  flex-direction: column;
  width: 70%;
  height: max-content;
  background-color: var(--color-main-background);

  &--tablet {
    width: 100%;
    padding-left: 0;
  }

  &__input {
    display: flex;
    flex-direction: column;
    height: max-content;
    padding-left: 20px;
    margin-bottom: 20px;
  }

  h1 {
    color: var(--color-header-font);
    font-weight: normal;
  }

  h2 {
    font-weight: normal;
  }
}

:export {
  block: $block;
  blockTablet: $blockTablet;
  blockInput: $blockInput;
}
