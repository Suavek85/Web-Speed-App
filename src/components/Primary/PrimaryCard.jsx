import React from 'react'
import classNames from 'classnames/bind'
import { useMediaQuery } from 'react-responsive'

import InputWrapper from './Input/InputWrapper'
import LighthouseWrapper from './Lighthouse/LighthouseWrapper'
import { tabletWidth } from'../../scss/mediaqueries'
import styles from './PrimaryCard.scss'

export default function PrimaryCard() {

  //STYLES
  const cx = classNames.bind(styles)
  const getClass = cx({
    block: true,
    blockTablet: useMediaQuery(tabletWidth),
  })

  return (
    <div className={ getClass } >
      <div className={ styles.headerInputWrapper } >
        <InputWrapper />
      </div>     
      <LighthouseWrapper />
    </div>
    
  )
}
