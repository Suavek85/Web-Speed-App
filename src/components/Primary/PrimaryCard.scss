@import '../../scss/styles.scss';

.primarycard__wrapper {
    width: 70%;
    height: max-content;
    display: flex;
    flex-direction: column;
    background-color: $color-main-background;
    &--tablet {
      width: 100%;
      padding-left: 0;
    }
    h1 { 
      color: cadetblue;
    }
}

.primarycard__header-input-wrapper {
    display: flex;
    flex-direction: column;
    padding-left: 20px;
    margin-bottom: 20px;
    height: max-content;
}

.primarycard__header {
    display: flex;
    align-items: center;
}

:export {
    block: primarycard__wrapper;
    blockTablet: primarycard__wrapper--tablet;
    headerInputWrapper: primarycard__header-input-wrapper;
  }

