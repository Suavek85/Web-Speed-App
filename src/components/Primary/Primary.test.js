import React from 'react'
import { shallow } from 'enzyme'
import Primary from './Primary'

it('Should match the snapshot', () => {
  const wrapper = shallow(<Primary />)
  expect(wrapper).toMatchSnapshot()
})