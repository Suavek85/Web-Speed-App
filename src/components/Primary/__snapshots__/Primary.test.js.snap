// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match the snapshot 1`] = `
<main
  className="block"
>
  <section>
    <InputWrapper />
  </section>
  <LightHouse />
</main>
`;
